cli2/morphir-json-schema-gen.ts

```diff
@@ -21,6 +21,7 @@ program
     .option('-g, --group-schema-by <string>', 'Group generate schema by package, module or type.', 'package')
     .option('-c, --use-config', 'Use configuration specified in the config file.', false)
     .option('-ls, --include <comma.separated,list.of,strings>', 'Limit what will be included.', '')
+    .option('-d, --use-decorators', 'Use configuration specified in the config file.', false)
```

Comment:
The description is confusing here. It doesn't even mention decorators. This should also be on by default.

Reply:
A more descriptive text has been added.